rpcd/playbooks/roles/elasticsearch/tasks/elasticsearch_post_install.yml

```diff
@@ -17,8 +17,8 @@
   template:
     src: "{{ item }}"
     dest: "/etc/elasticsearch/{{ item }}"
-    owner: "root"
-    group: "root"
+    owner: "elasticsearch"
```

Comment:
For now.....but the correct pattern as @sigmavirus24 suggested is to convert to role defaults.